src/components/authpage/authpage.css

```diff
@@ -0,0 +1,13 @@
+.auth-container{
+    align-items: center;
+    background-color: var(--OFF-WHITE);
+    border-radius: 1rem;
+    box-shadow: 0 5px 1rem 0 var(--LIGHT-GREY);
+    display: flex;
+    flex-direction: column;
+    flex-grow: 0;
+    justify-content: center;
+    max-width: 45rem;
```

Comment:
you can provide value of width in percentage

Reply:
actually I have provided already below. This is max-width.